index.bs

```diff
@@ -5416,6 +5422,30 @@ authenticator=] and [=client device=] SHOULD work together to ensure that the ex
 only to the operating system user that created that [=platform credential=].
 
 
+## Privacy of [PII] stored in authenticators ## {#sctn-pii-privacy}
+
+[=Authenticators=] MAY provide additional information to [=clients=] outside what's mandated by this specification, e.g.,
+to enable the [=client=] to provide a rich UI with which the user can pick which [=credential=] to use for an [=authentication
+ceremony=]. If an [=authenticator=] chooses to do so, it SHOULD NOT expose [PII] unless successful [=user verification=] has been
+performed. If the [=authenticator=] supports [=user verification=] with more than one concurrently enrolled user, the
+[=authenticator=] SHOULD NOT expose [PII] of users other than the currently [=user verified|verified=] user. Consequently, an
+[=authenticator=] that is not capable of [=user verification=] SHOULD NOT store [PII].
+
+For the purposes of this discussion, the [=user handle=] conveyed as the {{PublicKeyCredentialUserEntity/id}} member of
+{{PublicKeyCredentialUserEntity}} is not considered [PII].
+
+These recommendations serve to prevent an adversary with physical access to an [=authenticator=] from extracting [PII] about the
+owner(s) of the [=authenticator=].
```

Comment:
owner(s) of the [=authenticator=] =>  [=authenticator=] 's enrolled user(s)

..?

Reply:
Looks good, thanks!